src/batched/dense/KokkosBatched_Gemm_Decl.hpp

```diff
@@ -225,6 +225,12 @@ class BatchedSerialGemm;
 ///                             by tile sizes.
 ///                             BoundsCheck::Yes The functor will     perform bound checks (recommended)
 ///                             BoundsCheck::No  The functor will NOT perform bound checks
+/// \tparam ArgAlphaFmaTag      Specifies whether to apply alpha during fmas.
+///                             AlphaFmaTag::Yes alpha will be applied during fma (C = C * alpha + AB).
+///                             AlphaFmaTag::No  alpha will not be applied during fma (C = C + AB).
+/// \tparam ArgAlphaMulTag      Specifies whether to perform global memory access
+///                             AlphaFmaTag::Yes alpha will be applied during mul (A * B * alpha).
+///                             AlphaFmaTag::No  alpha will not be applied during mul (A * B).
```

Comment:
@e10harvey Why do we need two tags ```ArgAlphaFmaTag``` and ```ArgAlphaMulTag```? If not applying alpha during fmas, then it is applying alpha during mul. Can we just use one tag? Do I miss any logic here?